src/server/utils/validations.js

```diff
@@ -49,6 +49,14 @@ const validations = {
     }
     return validatedString;
   },
+
+  validateName: (data, count, nullable) => {
```

Comment:
is there another file/name that makes sense for this function? it isn't really validating the name as much as it is formatting it. maybe just add it to the enrollment system file as a function called `formatName` or similar? also, you aren't passing in the last 2 arguments to this function, so you could always pass in `nullable=false` to the `validateString` method, which will then always return a string and the `isEmpty` check could be removed from this function